recipes/paleomix/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "paleomix" %}
+{% set version = "1.2.14" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MikkelSchubert/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
Please use the version from PyPI. URL could be `https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz`